### PR TITLE
[PhpUnitBridge] Remove SetUpTearDownTrait

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/composer.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
     "require": {
         "php": ">=7.1.3 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
         "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
-        "php": ">=7.1.3",
-        "symfony/deprecation-contracts": "^2.1"
+        "php": ">=7.1.3"
     },
     "require-dev": {
+        "symfony/deprecation-contracts": "^2.1",
```
Comment (jderusse) [inline src/Symfony/Bridge/PhpUnit/composer.json:24]: hmmm whats for?

Comment (derrabus) [reply src/Symfony/Bridge/PhpUnit/composer.json:24]: `trigger_deprecation` is still used in tests of this component.

**Review by jderusse — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
(rebase needed)

**Comment by derrabus** [comment]
Done.

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**